queries/promql/highlights.scm

```diff
@@ -0,0 +1,47 @@
+; highlights.scm
+
+[
+  "\*"
```

Comment:
Is there any specific reason for escaped * here

Reply:
Ah, probably not; thanks! Was it necessary at some point? I think i remember that it was but dont know for sure!